packages/web-client/app/components/auth/index.ts

```diff
@@ -0,0 +1,92 @@
+import Component from '@glimmer/component';
+import Layer2Network from '@cardstack/web-client/services/layer2-network';
+import HubAuthentication from '@cardstack/web-client/services/hub-authentication';
+import UA from '@cardstack/web-client/services/ua';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { race, rawTimeout, task, TaskGenerator } from 'ember-concurrency';
+import { taskFor } from 'ember-concurrency-ts';
+import config from '@cardstack/web-client/config/environment';
+
+const AUTH_STEPS = {
```

Comment:
The other day I came across this article https://swizec.com/blog/stop-shouting-shouting/ and it struck a chord with me 😀 I know we use this pattern across the app but wondering what you think about simply using a type for this

Reply:
i think it's ok, but we have no types in templates.... so I'm shouting to get the point across to hbs

Reply:
If templates had types I would be happy to use types